backend/src/Edna/Setup.hs

```diff
@@ -0,0 +1,28 @@
+module Edna.Setup
+  ( EdnaContext (..)
+  , Edna
+  , edConfig
+  , runEdna
+  ) where
+
+import Universum
+
+import Lens.Micro.Platform (makeLenses)
+
+import Edna.Config.Definition (EdnaConfig(..))
+import RIO (RIO, runRIO)
+
+data EdnaContext = EdnaContext
+  { _edConfig :: !EdnaConfig
+  }
+
+makeLenses ''EdnaContext
+
+type Edna = RIO EdnaContext
```

Comment:
Minor naming suggestion: maybe let's call it `EdnaM`? That sounds more consistent with other projects I have worked on. But maybe that's not the case for your projects, so feel free to ignore. It can have another suffix, e. g. `EdnaT`, just to be more specific than `Edna`.

Reply:
So, it was called not by its haskell meaning but by common meaning\
But I can rename it if you want (with 🤔 about how many fixup commits I need to do it :peka:)